environments/liveness/liveness-environment/amplify/backend/function/livenessenvironment3477af97/src/package.json

```diff
@@ -6,7 +6,7 @@
   "type": "module",
   "license": "Apache-2.0",
   "dependencies": {
-    "@aws-sdk/client-rekognition": "latest"
+    "@aws-sdk/client-rekognition": "3.347.1"
```

Comment:
Question, how did this get updated?

Reply:
If it's set to latest do we need to change it?